aiosmtpd/docs/cli.rst

```diff
@@ -41,42 +41,8 @@ equivalent to the above ``python3`` invocation::
 Options
 =======
 
-Optional arguments include:
-
-``-h``, ``--help``
-    Show this help message and exit.
-
-``-n``, ``--nosetuid``
-    This program generally tries to setuid ``nobody``, unless this flag is
-    set.  The setuid call will fail if this program is not run as root (in
-    which case, use this flag).
-
-``-c CLASSPATH``, ``--class CLASSPATH``
-    Use the given class, as a Python dotted import path, as the :ref:`handler
-    class <handlers>` for SMTP events.  This class can process received
-    messages and do other actions during the SMTP dialog.  Uses a debugging
-    handler by default.
-
-``-s SIZE``, ``--size SIZE``
-    Restrict the total size of the incoming message to ``SIZE`` number of
-    bytes via the `RFC 1870`_ ``SIZE`` extension.  Defaults to 33554432 bytes.
-
-``-u``, ``--smtputf8``
-    Enable the SMTPUTF8 extension and behave as an `RFC 6531`_ SMTP proxy.
-
-``-d``, ``--debug``
-    Increase debugging output.
-
-``-l [HOST:PORT]``, ``--listen [HOST:PORT]``
-    Optional host and port to listen on.  If the ``PORT`` part is not given,
-    then port 8025 is used.  If only ``:PORT`` is given, then ``localhost`` is
-    used for the host name.  If neither are given, ``localhost:8025`` is used.
-
-Optional positional arguments provide additional arguments to the handler
-class constructor named in the ``--class`` option.  Provide as many of these
-as supported by the handler class's ``from_cli()`` class method, if provided.
-
+Optional arguments are described in the :ref:`man page <manpage>` document.
```

Comment:
Too many repeated info that needs to be synced with one another; let's go with a "Single Source of Truth".